exasol_script_languages_container_tool/lib/tasks/upload/language_def_parser.py

```diff
@@ -0,0 +1,118 @@
+from pathlib import PurePosixPath
+from typing import List, Tuple, Union
+from urllib.parse import parse_qs, urlparse
+
+from exasol_script_languages_container_tool.lib.models.language_activation import (
+    BuiltInLanguageDefinitionURL,
+    ChrootPath,
+    LanguageDefinitionURL,
+    SLCLanguage,
+    SLCParameter,
+    UdfClientAbsolutePath,
+    UdfClientRelativePath,
+)
+
+
+def _parse_builtin_language_definition(url: str) -> BuiltInLanguageDefinitionURL:
+    language = url.replace("builtin_", "")
+    try:
+        slc_language = next(
+            slc_language_enum
+            for slc_language_enum in SLCLanguage
+            if slc_language_enum.name.lower() == language.lower()
+        )
+    except StopIteration:
+        raise ValueError(f"Unknown builtin language: {url}")
+    return BuiltInLanguageDefinitionURL(language=slc_language)
+
+
+def _build_udf_client_abs_path_from_fragments(
+    fragment_parts: Tuple[str, ...]
+) -> UdfClientAbsolutePath:
+    if len(fragment_parts) < 3 or fragment_parts[0] != "buckets":
+        raise ValueError(
+            f"Expected format of the fragment in the URL for absolute udf client path is "
+            f"'/buckets/<bucketfs_name>/<bucket_name>/...' or"
+            f"'buckets/<bucketfs_name>/<bucket_name>/...' or"
+        )
+    udf_client_executable = None
+    if len(fragment_parts) > 3:
+        udf_client_executable = PurePosixPath("/".join(fragment_parts[3:]))
+
+    return UdfClientAbsolutePath(
+        bucketfs_name=fragment_parts[1],
+        bucket_name=fragment_parts[2],
+        executable=udf_client_executable,
+    )
+
+
+def _parse_udf_client_path(
+    fragment: str,
+) -> Union[UdfClientRelativePath, UdfClientAbsolutePath]:
+    fragment_path = PurePosixPath(fragment)
+    fragment_parts = fragment_path.parts
+
+    if len(fragment_parts) == 0:
+        return UdfClientRelativePath(executable=None)
+
+    if fragment_path.is_absolute():
+        fragment_parts = fragment_path.parts[1:]  # Remove leading "/"
+        return _build_udf_client_abs_path_from_fragments(fragment_parts)
```

Comment:
This we actually need to check, if a absolute path always need to start with buckets

Reply:
changed parsing here:
If the path does not start with `/buckets` or `buckets` -> Use original path as is.